elm.json

```diff
@@ -6,6 +6,7 @@
     "elm-version": "0.19.1",
     "dependencies": {
         "direct": {
+            "dillonkearns/elm-markdown": "7.0.1",
```

Comment:
I can't remember the benefit of this over the core library, but as long as there are not downsides, let's keep it.

Reply:
i pulled this from the TD dependency